ui/pages/notification-details/notification-details-footer/notification-details-footer.tsx

```diff
@@ -39,6 +39,11 @@ export const NotificationDetailsFooter = ({
           <footer.ExtensionLink notification={notification} />
         </>
       )}
+      {footer.type === 'footer_snap_notification' && (
+        <>
```

Comment:
This fragment is unnecessary.

Reply:
I thought so too, was just following the notification team's pattern 😅, I'll change it.